### DISCUSSION TITLE
Not being able to obtain inventory information for warehouses inside the country.

### DISCUSSION THREAD

**Discussion by VictorXefi** [amzn/selling-partner-api-models#2825] [General]
I'm attempting to obtain inventory reports for various USA warehouse locations, 
https://www.honouroceanshipping.com/locations-of-amazon-fulfillment-centers-in-usa/ 
I've looked from the Selling Partner API, but I can't seem to discover a way.

At the default inventory endpoint, the inventory is managed based on MarketplaceID for the entire USA country, and the warehouse location information is not retrieved. https://developer-docs.amazon.com/sp-api/docs/fba-inventory-api-v1-use-case-guide.

Exists an alternative endpoint where I can obtain warehouse location information for those inventory data?

**Comment by JohnMirro** [comment]
I want to know too, but about EU countries
